OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -162,7 +155,8 @@ public interface INotifyChat { bool OnChat(string from, string message); }
 	public interface INotifyProduction { void UnitProduced(Actor self, Actor other, CPos exit); }
 	public interface INotifyOtherProduction { void UnitProducedByOther(Actor self, Actor producer, Actor produced, string productionType, TypeDictionary init); }
 	public interface INotifyDelivery { void IncomingDelivery(Actor self); void Delivered(Actor self); }
-	public interface INotifyDocking { void Docked(Actor self, Actor harvester); void Undocked(Actor self, Actor harvester); }
+	public interface INotifyDock { void Docked(); void Undocked(); }
```

Comment:
Traits may need to know *what* they have docked with, in order to e.g. set appropriate artwork state or grant actor-specific conditions.

Reply:
Yes, these interfaces will grow in the future when BitSet docktypes get introduced as well as more interfaces merged. For now this is just redundant information

Reply:
More specifically `INotifyResupply` interface

Reply:
Made it inline with our current convention